src/List.js

```diff
@@ -1,11 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import {
+  collection,
+  getDocs,
+  query,
+  onSnapshot,
+  doc,
+  setDoc,
+} from '@firebase/firestore';
+import { db } from './lib/firebase.js';
 
 export function List() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
```

Comment:
In the Component App you copied and modified according to the issue the Useefect Hook From App.js to List.js here  and moved the method handleClick from App.js to the new AddForm, why didn't you delete this code on the App component?